libexec/rc/rc.d/dhclient

```diff
@@ -48,6 +48,10 @@ dhclient_prestart()
 		rc_flags="${rc_flags} -b"
 	fi
 
+	dhclient_arpwait=$(get_if_var $ifn dhclient_arpwait_IF $dhclient_arpwait)
+	if checkyesno dhclient_arpwait; then
```

Comment:
and this is now backwards; we need to add `-n` if `dhclient_arpwait` is set to *no*.